cmd/main.go

```
@@ -0,0 +1,88 @@
+			// 기존 식단 조회 API
+			api.GET("/restaurants/:id", mealHandler.GetRestaurantMeals)
+			
+			// // 엑셀 처리 API
+			// api.POST("/upload/excel", excelHandler.UploadAndProcessExcel)
+			api.POST("/process/excel/local", excelHandler.ProcessLocalExcel) // 개발용
+	}
+	// Set up Swagger
+	router.GET("/swagger/*any", ginSwagger.WrapHandler(swaggerFiles.Handler))
+
+	// Run the server
+	router.Run(":8080")
```

golangci-lint (1.64.8)

Error return value of `router.Run` is not checked

(errcheck)